src/en/proposals/emogarbage-xenobio.md

```diff
@@ -0,0 +1,108 @@
+
+By default, you'll be supplied with a crate of monkeys and some N2O. Knock a monkey out, inject it with the cocktail, and then stick it inside one of the various **Growing Vats** scattered inside of the xenobio lab.
+Once inside, you simply need to wait for the mutant to grow, watching from afar.
+When finished growing, use a plunger to pull it out.
+
+If you pull it out prematurely, you may simply get an unmutated monkey or just a pile of slop. If you leave it in too long, the mutant may wake up and become angry, destroying the vat as it escapes.
+
+Growing Vats must be stocked with a decent level of nutriment and saline in order to facilitate the mutation process. 
+Failure to do so may lead to your mutant coming out dead.
+
+Whatever the case may be: you've now grown a mutant.
+Get it inside of a chamber and start experimenting.
+
+## Mutant Care and Growth
+You have a disgusting mutant in a cage: now what?
+
+Your main goal is to care for the mutant in order to help it grow.
+This requires satisfying certain conditions for it. Primarily:
+- Food: vegetables, meat, live animals, etc.
+- Temperature: Average, cold, warm, boiling
+- Pressure: Average, low, or high
+- Enrichment: Toys, playmates, prey, entertainment.
+
+When you satisfy these conditions, the mutant slowly grows.
+Making sure that these conditions stay constant is important, as mutants will react more strongly when they have a condition satisfied and then it goes away.
+
+But what happens when you don't satisfy these conditions?
+In short: bad things.
+Improper diet and environment may cause the mutant to weaken and slowly die, preventing you from researching it properly.
+Uncomfortable or unsatisfied mutants may additionally grow aggressive, lashing out at their cage and researchers.
+
+This may cause them to break out and escape their cages, at which point they may ravage all of science or even escape out into the station.
+Keeping them happy should (mostly) prevent this, however, so it's essential to do so.
+
+## Research
+Once a mutant has finished its growth, it's appearance will change and scientists will be able to conduct research.
+This is when points are generated.
+
+First, the scientists must figure out how to kill the mutant.
+Gassing the chamber with N2O is a smart bet but it may come down to lasers or even good ol' fashioned hand-to-hand combat.
+Whatever the case, it must be dead.
+
+Then the corpse can be taken to the autopsy station and a post-mortem analysis can be done.
+This is simply done with a large scanner machine while surgery isn't implemented. 
+After the machine finishes the post-mortem, research will be added to the server and the corpse can be disposed of.
+
+Research is awarded based on the instability of the creature as well as the number of splices in the genome.
+However, the research is only rewarded for a unique genome.
+Attempting to perform autopsies on a creature with the same genome multiple times will not result in additional points.
```

Comment:
To prevent extra value from combining same genomes over and over again with very small and not daring changes there should be system of giving points per unique genome combinations (not yet dissected).